nrrd/tests/test_reading.py

```diff
@@ -100,6 +100,26 @@ def test_read_raw_header(self):
         header = nrrd.read_header(('NRRD0005', 'my extra info:=my : colon-separated : values'))
         np.testing.assert_equal(expected_header, header)
 
+    def test_read_dup_field_error_and_warn(self):
+        expected_header = {u'type': 'float', u'dimension': 3}
+        header_txt_tuple = ('NRRD0005', 'type: float', 'dimension: 3', 'type: float')
+
+        dup_message = "Duplicate header field: 'type'"
```

Comment:
Along with comment below, would remove dup_message and just put as string in *assertRaisesRegex* function